src/backends/beamcoder.ts

```diff
@@ -239,7 +239,7 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
      */
     async getFrameAtTime(targetTime: number): Promise<beamcoder.Frame> {
         VERBOSE && console.log(`getFrameAtTime time(s)=${targetTime}`);
-        const targetPts = Math.floor(this._timeToPTS(targetTime));
+        const targetPts = Math.round(this._timeToPTS(targetTime));
```

Comment:
With Math.floor it seeks it 60, 60, 62 frames

Reply:
And now we get 60, 61 and 62

Reply:
Nice find!